compiler/lineinfos.nim

```diff
@@ -67,6 +67,7 @@ type
     warnResultUsed = "ResultUsed",
     warnCannotOpen = "CannotOpen",
     warnFileChanged = "FileChanged",
+    warnDuplicateModuleImport = "DuplicateModuleImport",
```

Comment:
If at all this should be a hint, not a warning.

Reply:
=> doing this in https://github.com/nim-lang/Nim/pull/18366